src/sidebar/store/modules/session.js

```diff
@@ -23,8 +23,16 @@ const initialProfile = {
   userid: null,
 };
 
-function init() {
+function init(settings) {
   return {
+    /**
+     * The app's authentication domain, from settings
+     * FIXME: This returns an empty string when `authDomain` is missing
+     * because other app logic has long assumed its string-y presence:
+     * behavior when it's missing is undefined. This setting should be
+     * enforced similarly to how `apiUrl` is enforced.
+     */
+    authDomain: settings?.authDomain ?? '',
```

Comment:
I suspect the meaning of "authentication domain" won't be very obvious to someone who doesn't know "h" well. Especially because almost everywhere else we talk about the _authority_ that users belong to. Perhaps we should call this `defaultAuthority` or `firstPartyAuthority` and explain briefly how it relates to users (profiles, annotations) that the app will encounter.